test/compliance_tool/test_compliance_check_aasx.py

```diff
@@ -6,12 +6,36 @@
 # SPDX-License-Identifier: MIT
 import os
 import unittest
+import zipfile
+
+import pyecma376_2
```

Comment:
Why is this import added? It doesn't seem to be used.

Reply:
Removed in https://github.com/eclipse-basyx/basyx-python-sdk/pull/143/commits/923c14a85a4ef6f9a7ca45bd4737fc5aa7adb828